app/models/concerns/current_tracked_by_changeset.rb

```diff
@@ -105,24 +106,27 @@ def changeable_attributes
         reflections.values.map(&:foreign_key).map(&:to_sym) -
         [:id, :created_at, :updated_at, :version]
       )
+      @changeable_attributes -= @sticky_attributes if sticky
+      @changeable_attributes
     end
 
     private
 
-    def current_tracked_by_changeset(kind_of_model_tracked: nil, virtual_attributes: [], protected_attributes: [])
+    def current_tracked_by_changeset(kind_of_model_tracked: nil, virtual_attributes: [], protected_attributes: [], sticky_attributes: [])
```

Comment:
Here is where logic will be done to subtract object-level attribute values that have been edited.